.pre-commit-config.yaml

```diff
@@ -48,8 +48,15 @@ repos:
     - id: pyproject-fmt
       # https://pyproject-fmt.readthedocs.io/en/latest/#calculating-max-supported-python-version
       additional_dependencies: ["tox>=4.9"]
--   repo: local
-    hooks:
+- repo: local
+  hooks:
+    - id: pylint
```

Comment:
Can you align this with the entries below? (I was a bit confused at first)

Reply:
Done :) I could add prettier to fix that automatically in the future. The installation step in pre-commit is pretty lengthy (only one time though).